index.html

```diff
@@ -269,7 +269,7 @@ <h3 class="about-myself-connect">LET'S CONNECT</h3>
           </div>
         </div>
       </section>
-      <section class="contact-card-section">
+      <form class="contact-card-section">
         <div class="all-components">
```

Comment:
- [ ] Form should use the action and method attribute to integrate with formspree service.
- action value should be similar to https://formspree.io/f/{form_id}
- method value equals POST
> please follow the link https://formspree.io/html, to learn more about the formspree and use it in your project